docs/compilers/CSharp/testplans/stackalloc_initializers.md

```diff
@@ -10,24 +10,24 @@ In such case we can just check them off.
 ### correctness ###
 - [ ] check that partial result is not visible. - if element throws and exception caught, whole thing is not reassigned to the new value. (try spans and ordinary stackallocs)
 - [ ] if we refer to the elements of outer local in the initializers we still see the old values.
-- [ ] put `await` in the size or in the middle of element initializers
+- [x] put `await` in the size or in the middle of element initializers
 - [ ] initializer uses a local that is captured by a lambda expression. See that if lambda changes the value of the local, then we see the updated value when initializing
 
 ### errors ##
 - [x] invalid array shapes - multidimensional, nested, size mismatches, ... 
 - [x] bad conversions
 - [x] bad local initialization contexts - using, fixed. 
-- [ ] should it work in `for(int* x = stackalloc...)`, probably the same ways as before... ?  
+- [x] should it work in `for(int* x = stackalloc...)`, probably the same ways as before... ?  
 - [X] missing various span parts.
-- [ ] outer expression is not usable in elements (not assigned yet)
+- [x] outer expression is not usable in elements (not assigned yet)
 - [ ] in the array type inference case the outer expression cannot be used to determine the type - [ ] should be an error and should not be some kind of crash due to circular dependency
 - [ ] make it to infer strange types. 
-	- [ ] dynamic
+	- [x] dynamic
 	- [ ] ref-struct
```

Comment:
The the exists but doesn't fail due to https://github.com/dotnet/roslyn/issues/25086

Reply:
We should sync the branch after the fix https://github.com/dotnet/roslyn/pull/25131 goes in. Then we can assert the failure as it is expected.

Reply:
That's ok. and can be marked as having coverage. The test could be "skipped" for now - due to [https://github.com/dotnet/roslyn/issues/25086](https://github.com/dotnet/roslyn/issues/25086)

---
In reply to: [171064165](https://github.com/dotnet/roslyn/pull/25101#discussion_r171064165) [](ancestors = 171064165)

Reply:
The branch is severely behind `features/compiler` if that's alright, I'll extract this test to a skipped one.

Reply:
I think it is ok as-is. The test is now passing but has a comment that it will fail when merged.

---
In reply to: [171702476](https://github.com/dotnet/roslyn/pull/25101#discussion_r171702476) [](ancestors = 171702476)

Reply:
I can merge the /compiler, just did not want to interfere with your work. Perhaps will do after this PR?

---
In reply to: [171712095](https://github.com/dotnet/roslyn/pull/25101#discussion_r171712095) [](ancestors = 171712095,171702476)